modules/local/build_intervals/main.nf

```diff
@@ -1,6 +1,5 @@
 process BUILD_INTERVALS {
     tag "$fasta_fai"
-    label 'process_medium'
```

Comment:
what are the default labels?

Reply:
per default it should now go to 1 cpu, those at least are the default values in the base config